cli/main.c

```diff
@@ -1924,10 +1924,35 @@ static struct prog_info prog_info = {
 		"(ex: /dev/switchtec0)",
 };
 
+#ifndef _WIN32
+static void sig_handler(int signum)
+{
+	if (signum == SIGBUS) {
+		fprintf(stderr, "Device returns are unexpected, "
+			"please check the setup.\n");
```

Comment:
Can we use a better error message. Something like:

"Error communicating with the device. Please check your setup."